CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 - Fixes a issue where `darkreader-fallback` wasn't removed from the DOM, when Dark Reader finds a `<meta name="darkreader-lock">` element.
 - Be stricter when the user specifies a last slash for a URL in the sitelist.
 - Display a warning that "System" Automation might not work properly in Chromium on Linux and Kiwi on Android.
-- Improve Kiwi browser support: work around for Kiwi bug in file download during settings import, fix opening open Dark Reader DevTools
+- Improve Kiwi browser support: work around for Kiwi bug in file download during settings import, fix opening open Dark Reader DevTools.
+- Add delete all button on site list.
```

Comment:
```suggestion
- Add a new button to the site list page (new UI), which sits on the top-left corner of the list. It will clear out the site list.
```